conan/tools/scm/git.py

```diff
@@ -27,6 +28,12 @@ def get_commit(self):
         except Exception as e:
             raise ConanException("Unable to get git commit in '%s': %s" % (self.folder, str(e)))
 
+    def get_version(self):
+        try:
+            return Version(self._run('describe --always --tags --abbrev=9'))
```

Comment:
The 9 magic number seems a bit arbitray, isn't it? Maybe it is a good common practice for git? Users might want another lenght?

Reply:
Yes, it's arbitray, you're right. Github UI, for example, shows 7 symbols of the last cache